modules/telco-ran-ptp-operator.adoc

```diff
@@ -7,9 +7,9 @@
 = PTP Operator
 
 New in this release::
-* https://issues.redhat.com/browse/CNF-12504[Enable 2 Port Ordinary Clock Use Case for IA]
-* https://issues.redhat.com/browse/CNF-13189[CNF-13189: Remove the internal API and sidecar support]
-* https://issues.redhat.com/browse/CNF-16651[CNF-16651: 3 Card WPC T-G]
+* 2 Port Ordinary Clock Use Case for IA is enabled.
+* The PTP events REST API v1 and events consumer application sidecar support are removed.
+* A maximum of 3 Westport channel NIC configurations is now supported for T-GM.
```

Comment:
Noting that in line 12 the numeral 3 is used. In lines 19 and 20, words for numbers are used. (I know you did not update lines 19 and 20.) Perhaps use the same convention for consistency.